tilequeue/process.py

```diff
@@ -65,12 +66,18 @@ def _preprocess_data(feature_layers, shape_padded_bounds):
 # of other layers (e.g: projecting attributes, deleting hidden
 # features, etc...)
 def _postprocess_data(feature_layers, post_process_data,
-                      zoom):
+                      zoom, bounds):
 
     for step in post_process_data:
         fn = loadClassPath(step['fn_name'])
         params = step['params']
 
+        # inspect the function to tell if it has kwarg called 'bounds'
+        # and, if so, update the params to include the tile coordinate.
+        if 'bounds' in getargspec(fn).args:
+            params = params.copy()
+            params['bounds'] = bounds
+
```

Comment:
We might want to explicitly just pass bounds to every single function, or grow a context/state/env variable that has got it in it and always pass everything along there.